ui/pages/confirmations/confirmation/templates/add-ethereum-chain.js

```diff
@@ -371,6 +373,13 @@ function getValues(pendingApproval, t, actions, history, data) {
             [t('blockExplorerUrl')]: t('blockExplorerUrlDefinition'),
           },
           warnings: {
+            [t('networkURL')]:
+              pendingApproval.requestData.rpcUrl ===
+              punycode.toASCII(pendingApproval.requestData.rpcUrl)
+                ? undefined
+                : t('networkUrlErrorWarning', [
+                    pendingApproval.requestData.rpcUrl,
```

Comment:
Minor, but as we reference the full property chain a few times, could we define variables above such as `rpcUrl` and `rpcUrlAscii`?